src/helpers/api.ts

```diff
@@ -0,0 +1,22 @@
+const API_URL_USERS = 'https://jsonplaceholder.typicode.com/users';
+const API_URL_TODOS = 'https://jsonplaceholder.typicode.com/todos';
```

Comment:
or you could have `https://jsonplaceholder.typicode.com` as a base URL and add `/users` and so on where it is needed